README.md

```diff
@@ -23,37 +17,26 @@ composer.json
 
 ```json
 {
-    "name": "my-project",
-    "version": "1.0.0",
     "require": {
         "fluent/logger": "v0.3.7"
     }
 }
 ```
 
-````
-wget http://getcomposer.org/composer.phar
-php -d detect_unicode=0 composer.phar install
-````
+# Backward Compatibility CHnages
 
-### copy directory
+As of v4, all loggers but `FluentLogger` are removed.
```

Comment:
In this meaning, I think that "all loggers **except for** `FluentLogger` are removed" is more suitable.